### ISSUE TITLE
Do we need CEFLAG?

### ISSUE THREAD

**Issue by scottcoughlin2014** [COSMIC-PopSynth/COSMIC#163]
So this is the only place `CEFLAG` (or `ceflag`) is called
```
cosmic/src/comenv.f:111:         IF(CEFLAG.NE.3) EORBI = MC1*MC2/(2.D0*SEP)
cosmic/src/comenv.f:113:         IF(CEFLAG.NE.3) EORBI = MC1*M2/(2.D0*SEP)
```
This seems like we do not need this "flag".

Scotty

**Comment by scottcoughlin2014** [comment]
@chase-kimball we discussed checking whether this mythical `de Kool` is a real thing or not

Scotty

**Comment by chase-kimball** [comment]
The de Kool method sets a binding energy of M1*M2/2*sep. This is consistent with equation 10 (second term on RHS) http://adsabs.harvard.edu/cgi-bin/nph-data_query?bibcode=1990ApJ...358..189D&link_type=ARTICLE&db_key=AST&high=#page=3

**Comment by michaelzevin** [comment]
This is addressed and fixed in #168
